src/util/user_database.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+
+user_database_connection = sqlite3.connect('users.db')
```

Comment:
database path should be configurable (in config.py)

Reply:
This file is non-persistent